setup.py

```diff
@@ -1,23 +1,41 @@
+import os
 from setuptools import setup, find_packages
 
-VERSION = '0.0.3' 
+VERSION = '0.0.3'
```

Comment:
Note: DO NOT REDEPLOY untile we bump the version number. Separate PR.

Reply:
:+1:

Reply:
redeploy is also waiting on transition of Exclosure domains to the company cloud acct.